frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -58,12 +58,30 @@ export const getAssociatedNodes = (pvs: K8sResourceKind[]): string[] => {
   return Array.from(nodes);
 };
 
+export const shouldDeployInternalAsMinimal = (nodes: NodeKind[]) => {
```

Comment:
```suggestion
export const DeployCloudAsMinimal = (nodes: NodeKind[]) => {
```

Reply:
you can deploy internal mode in VMWare as well.

Reply:
It's not a react component. (first character no need to be uppercase)